src/lib/abacus/dydxChainTransactions.ts

```diff
@@ -40,8 +40,8 @@ import { DydxAddress } from '@/constants/wallets';
 import { type RootStore } from '@/state/_store';
 // TODO Fix cycle
 // eslint-disable-next-line import/no-cycle
-import { placeOrderTimeout } from '@/state/account';
```

Comment:
nit: does the cycle still exist? (I'm surprised that there's no error after moving this to be after `setInitializationError`